permissions/src/desktopMain/kotlin/dev/icerock/moko/permissions/impl.kt

```diff
@@ -0,0 +1,25 @@
+package dev.icerock.moko.permissions
+
+class PermissionsControllerImpl : PermissionsController {
+    override suspend fun providePermission(permission: Permission) {
+        //TODO("Not yet implemented")
+    }
+
+    override suspend fun isPermissionGranted(permission: Permission): Boolean {
+        return true
+       // TODO("Not yet implemented")
+    }
+
+    override suspend fun getPermissionState(permission: Permission): PermissionState {
+        return PermissionState.Granted
```

Comment:
it's not true. on macos for example we have runtime permissions and JVM desktop should work with it too